docs/argo/latest/howtos/configure-argo-rollouts.md

```diff
@@ -95,7 +95,7 @@ metadata:
 
 ## 4. Apply the manifests in your cluster
 
-From your root of your locally forked rollouts-demo repository, apply the Kubernetes manifests to your cluster:
+From your root of your locally forked [rollouts-demo](https://github.com/datawire/rollouts-demo) repository, update the a8r.io/repository annotation from `git@github.com:datawire/rollouts-demo.git` to `git@gthub.com:<org>/<repo>.git`, then apply the Kubernetes manifests to your cluster with the command:
```

Comment:
```suggestion
From the root of your locally forked [rollouts-demo](https://github.com/datawire/rollouts-demo) repository, update the a8r.io/repository annotation from `git@github.com:datawire/rollouts-demo.git` to `git@gthub.com:<org>/<repo>.git`, then apply the Kubernetes manifests to your cluster with the command:
```

Reply:
Maybe adding something about the namespace where the things are created ?

kubectl apply -f ./manifests will create everything in the current selected namespace, usually default for a new cluster.